cryptoapi_windows.go

```diff
@@ -9,16 +9,60 @@ import (
 	"time"
 
 	"golang.org/x/sys/windows/registry"
+	"gopkg.in/hlandau/easyconfig.v1/cflag"
+)
+
+var (
+	cryptoApiFlagGroup            = cflag.NewGroup(flagGroup, "capi")
+	cryptoApiFlagLogicalStoreName = cflag.String(cryptoApiFlagGroup, "store", "Root",
+		"Name of CryptoAPI logical store to inject certificate into. Consider: Root, Trust, CA, My")
+	cryptoApiFlagLogicalStoreScope = cflag.String(cryptoApiFlagGroup, "scope", "HKLM",
+		"Scope of CryptoAPI certificate store. Valid choices: HKLM, HKCU, machine, user")
```

Comment:
Generally, I would suggest replacing this variable with one that accepts fixed strings like `current-user`, `system`, `enterprise`, and `group-policy`, and chooses the physical store via a 1-to-1 mapping of those strings to registry keys.

Reply:
As a follow-up PR, we could add other flags that accept a username (for the HKU hive), service name, or full physical store registry key path.